### PR TITLE
CHANGE:: SpuDTO

### PR DESCRIPTION
**Pull request by 10102004tan**
change Type long to Long, int to Integer

### CODE REVIEW

File: front-end/src/components/Pagination/index.jsx
Comment (tranhieuphuc12) [inline front-end/src/components/Pagination/index.jsx:1]: conflict hasn't been resolved yet

File: front-end/src/contexts/ProductProvider.jsx
Comment (tranhieuphuc12) [inline front-end/src/contexts/ProductProvider.jsx:1]: conflict hasn't been resolved yet

**Review by tranhieuphuc12 — CHANGES_REQUESTED** [review]

File: front-end/src/contexts/ProductProvider.jsx
Comment (10102004tan) [reply front-end/src/contexts/ProductProvider.jsx:1]: ok

**Review by chiendevj — APPROVED** [review]
done

**Review by DoNgocHieu — APPROVED** [review]
oke done

**Review by Datto098 — APPROVED** [review]
OK

**Review by tranhieuphuc12 — APPROVED** [review]

**Pull request merged**